src/include/daos_srv/pool.h

```diff
@@ -171,6 +171,17 @@ struct ds_pool_child {
 	void			*spc_metrics[DAOS_NR_MODULE];
 };
 
+struct svc_op_key {
```

Comment:
[Nit] Strictly speaking, we should name this `ds_pool_svc_op_key` (i.e., with the `ds_pool_` prefix).

Reply:
OK, makes sense. Since it is to be used by both pool and container perhaps I kept it non-specific with that in mind. I can make the adjustment in the next patch.